llvm/lib/TargetParser/RISCVTargetParser.cpp

```diff
@@ -119,6 +119,38 @@ void getFeaturesForCPU(StringRef CPU,
     else
       EnabledFeatures.push_back(F.substr(1));
 }
+
+namespace RISCVExtensionBitmaskTable {
+#define GET_RISCVExtensionBitmaskTable_IMPL
+#include "llvm/TargetParser/RISCVTargetParserDef.inc"
+
+} // namespace RISCVExtensionBitmaskTable
+
+namespace {
+struct LessExtName {
+  bool operator()(const RISCVExtensionBitmaskTable::RISCVExtensionBitmask &LHS,
```

Comment:
Can we just use 1 operator here
```
bool operator()(const RISCVExtensionBitmaskTable::RISCVExtensionBitmask &LHS,
                           const RISCVExtensionBitmaskTable::RISCVExtensionBitmask &RHS)
```
and wrap the `ExtName` in `getExtensionBitmask` with `RISCVExtensionBitmask`?

Reply:
I think lower::bound only requires one of the operators.